README.md

```diff
@@ -1,33 +1,60 @@
-# bioio
+# BioIO
 
 [![Build Status](https://github.com/bioio-devs/bioio/actions/workflows/ci.yml/badge.svg)](https://github.com/bioio-devs/bioio/actions)
 [![Documentation](https://github.com/bioio-devs/bioio/actions/workflows/docs.yml/badge.svg)](https://bioio-devs.github.io/bioio)
 [![PyPI version](https://badge.fury.io/py/bioio.svg)](https://badge.fury.io/py/bioio)
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![Python 3.6](https://img.shields.io/badge/python-3.9,3.10,3.11-blue.svg)](https://www.python.org/downloads/release/python-360/)
 
-
-Image reading, metadata management, and image writing for Microscopy images in Python
+Image Reading, Metadata Conversion, and Image Writing for Microscopy Images in Pure Python
 
 ---
 
-## Installation
-
-**Stable Release:** `pip install bioio`<br>
-**Development Head:** `pip install git+https://github.com/bioio-devs/bioio.git`
+## Documentation
 
-## Quickstart
+[See the full documentation on our GitHub pages site](https://bioio-devs.github.io/bioio/OVERVIEW.html)
 
-```python
-from bioio import example
+## Example Usage (see full documentation for more examples)
 
-print(example.str_len("hello"))  # prints 5
-```
+Assumptions for the below example ([if untrue, see full documentation for more examples](https://bioio-devs.github.io/bioio/OVERVIEW.html)):
+* File is an OME TIFF
+    * BioIO supports additional file types, this example is specific to OME TIFF
+* File fits in memory
+    * BioIO can work with images that cannot be stored in memory, this example is specific to an image that does fit in memory
```

Comment:
For brevity and simplicity, I wonder if these assumptions are really needed here.

Reply:
Yeah as you pointed out elsewhere I've repeated this is an example a view times so I'll remove